File: HW2/hw2.cpp
#include "Simulator.h"
#include <string>

int main() {
    Simulator sim;
    for(int i = 1 ; i < 7 ; i++){
        std::string index = std::to_string(i);
        sim.loadMemory("input/input"+index); 
        sim.simulate();
        sim.storeMemory("output/output"+index);
    }
    sim.loadAssembly("input/bonus.txt");
    sim.simulate();
    sim.storeMemory("output/bonus");
    return 0;
}

File: HW2/hw2.py
import Simulator

def main():
    sim = Simulator.Simulator()
    for i in range(1,6):
        sim.loadMemory("input/input{}".format(i))
        sim.simulate()
        sim.storeMemory("output/output{}".format(i))
    sim.loadAssembly("input/bonus.txt")
    sim.simulate()
    sim.storeMemory("output/bonus")


if __name__ == '__main__':
    main()
